Source/Demos/Demo.SpriteSheetAnimations/Game1.cs

```diff
@@ -28,13 +28,11 @@ public Game1()
             _graphicsDeviceManager = new GraphicsDeviceManager(this);
             Content.RootDirectory = "Content";
             IsMouseVisible = true;
-            Window.Title = $"MonoGame.Extended - {GetType().Name}";
-            Window.Position = Point.Zero;
```

Comment:
Please don't remove this code. It's there because Mac's have some issues with the default window position. I'm not a fan of placing the window in the top left corner either, but it's not hurting anything.